package.json

```diff
@@ -13,21 +13,21 @@
   "license": "Talk to the Legal Department",
   "main": "NestHydrationJS.js",
   "scripts": {
-    "coverage": "node_modules/istanbul/lib/cli.js cover node_modules/jasmine/bin/jasmine.js",
```

Comment:
I couldn't get it to error locally, but this change isn't necessary so I removed it.